lib/jxl/modular/encoding/encoding.cc

```diff
@@ -516,23 +516,20 @@ Status ModularDecode(BitReader *br, Image &image, GroupHeader &header,
   const std::vector<uint8_t> *context_map = &context_map_storage;
   if (!header.use_global_tree) {
     size_t max_tree_size = 1024;
```

Comment:
this could be uint64_t and then we don't even need if (max_tree_size < pixels ...., and at the end we could have the std::min as before

Reply:
Yes, that also works and is even simpler.